clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -159,7 +163,7 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
                                                     metadata: metadata,
                                                     tags: tags,
                                                     scheduling: scheduling,
-                                                    serviceAccounts: [serviceAccount])
+                                                    serviceAccounts: serviceAccount ? [serviceAccount] : [])
```

Comment:
It might be cleaner if `buildServiceAccount` were instead `buildServiceAccounts` and returned and empty list instead of null (which causes you to then have to do this jazz)